Jenkinsfile

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env groovy
+
+/**
+ * Build Switchcounter Workshop
+ *
+ */
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '30')),
+    parameters([
+        stringParam(defaultValue: 'http://downloads.creatordev.io/creator-openwrt/v1/pistachio/marduk/creator-sdk-1.tar.bz2',
```

Comment:
The SDK didn't change between creator release v1 and v2, so the squad chose not to put out a fresh version/copy in the creator v2 directory.